pkg/liqoctl/wait/wait.go

```diff
@@ -279,3 +280,52 @@ func (w *Waiter) ForGatewayClientSecretRef(ctx context.Context, gwClient *networ
 	s.Success("Gateway client Secret created successfully")
 	return nil
 }
+
+// ForConnection waits until the Connection resource has been created.
+func (w *Waiter) ForConnection(ctx context.Context, namespace string,
+	remoteCluster *discoveryv1alpha1.ClusterIdentity) (*networkingv1alpha1.Connection, error) {
+	s := w.Printer.StartSpinner("Waiting for Connection to be created")
+	var conn *networkingv1alpha1.Connection
+	err := wait.PollUntilContextCancel(ctx, 1*time.Second, true, func(ctx context.Context) (done bool, err error) {
+		var connections networkingv1alpha1.ConnectionList
+		clusterIDSelector := client.MatchingLabels{consts.RemoteClusterID: remoteCluster.ClusterID}
+		err = w.CRClient.List(ctx, &connections, client.InNamespace(namespace), clusterIDSelector)
+		if err != nil {
+			return false, client.IgnoreNotFound(err)
+		}
```

Comment:
Can you move this in pkg/utils/getters/k8sGetters.go and add the labelSelector in pkg/utils/labels/labelSelectors.go ? Do it only if it is done in other liqoctl waiting functions.